caikit/interfaces/nlp/tasks.py

```diff
@@ -24,7 +24,7 @@
 
 
 @task(
-    unary_parameters={"inputs": str},
+    unary_parameters={"text": str},
```

Comment:
Why are we making this change? Is this just the fact that it's more meaningful? This is kind of a larger question of how we're handling input arguments in the HTTP server versus gRPC. We went the route of bundling everything under `inputs` in the HTTP server to avoid needing to have all modules change, but if we're writing net-new modules, we could (should?) use `inputs` explicitly so that the gRPC objects line up too